tools/clang/lib/CodeGen/CGHLSLMS.cpp

```diff
@@ -572,26 +572,15 @@ static unsigned AlignBaseOffset(unsigned baseOffset, unsigned size,
   bool needNewAlign = Ty->isArrayType();
 
   if (IsHLSLMatType(Ty)) {
-    bool bColMajor = !bDefaultRowMajor;
-    if (const AttributedType *AT = dyn_cast<AttributedType>(Ty)) {
```

Comment:
I fixed this but filed #1721 as this code apparently supports inheriting from matrices??